packages/tooltips/src/styled/StyledTooltip.ts

```diff
@@ -5,78 +5,128 @@
     }
   }
 
-  return 'margin: 6px';
+  return css`
+    margin: ${margin};
+    border-radius: ${borderRadius};
+    padding: ${padding};
+    max-width: ${maxWidth};
+    line-height: ${lineHeight};
+    white-space: ${whiteSpace};
+    /*  Manipulate arrow sizing (which is based on ems). */
+    font-size: ${fontSize};
+    font-weight: ${theme.fontWeights.regular};
+    overflow-wrap: ${overflowWrap};
+
+    ${StyledParagraph} {
+      margin-top: ${paragraphMarginTop};
+    }
+
+    ${StyledTitle} {
+      display: ${titleDisplay};
+    }
+  `;
 };
 
-export interface IStyledTooltipProps {
-  hasArrow?: boolean;
-  size?: TOOLTIP_SIZE;
-  /** All valid [Popper.JS Placements](https://popper.js.org/popper-documentation.html#Popper.placements) */
-  placement?: POPPER_PLACEMENT;
-}
+const colorStyles = ({ theme }: ThemeProps<DefaultTheme>) => css`
+  box-shadow: 0 ${theme.space.base}px ${theme.space.base * 2}px 0
+    ${getColor('chromeHue', 600, theme, 0.15)};
+  background-color: ${getColor('chromeHue', 700, theme)};
+  color: ${theme.colors.background};
+`;
 
 /**
  * Accepts all `<div>` props
  */
 export const StyledTooltip = styled.div.attrs<IStyledTooltipProps>(props => ({
   'data-garden-id': COMPONENT_ID,
   'data-garden-version': PACKAGE_VERSION,
-  className: classNames(TooltipStyles['c-tooltip'], {
-    // Size
-    [TooltipStyles['c-tooltip--medium']]: props.size === 'medium',
-    [TooltipStyles['c-tooltip--large']]: props.size === 'large',
-    [TooltipStyles['c-tooltip--extra-large']]: props.size === 'extra-large',
-
-    // Arrows
-    [TooltipStyles['c-arrow']]: shouldShowArrow(props),
-    [ArrowStyles['c-arrow']]: shouldShowArrow(props),
-    [ArrowStyles['c-arrow--r']]: props.placement === 'left',
-    [ArrowStyles['c-arrow--rt']]: props.placement === 'left-start',
-    [ArrowStyles['c-arrow--rb']]: props.placement === 'left-end',
-    [ArrowStyles['c-arrow--b']]: props.placement === 'top',
-    [ArrowStyles['c-arrow--bl']]: props.placement === 'top-start',
-    [ArrowStyles['c-arrow--br']]: props.placement === 'top-end',
-    [ArrowStyles['c-arrow--l']]: props.placement === 'right',
-    [ArrowStyles['c-arrow--lt']]: props.placement === 'right-start',
-    [ArrowStyles['c-arrow--lb']]: props.placement === 'right-end',
-    [ArrowStyles['c-arrow--t']]: props.placement === 'bottom',
-    [ArrowStyles['c-arrow--tl']]: props.placement === 'bottom-start',
-    [ArrowStyles['c-arrow--tr']]: props.placement === 'bottom-end',
-
-    // RTL
-    [TooltipStyles['is-rtl']]: isRtl(props)
-  })
+  'data-placement': props.placement
 }))<IStyledTooltipProps>`
-  ${props => retrieveTooltipMargin(props)};
+  display: inline-block;
+  box-sizing: border-box;
+  direction: ${props => props.theme.rtl && 'rtl'};
+
+  ${colorStyles};
+  ${sizeStyles};
+  ${arrowStyles};
+
+  &[aria-hidden='true'] {
+    display: none;
+  }
+
   ${props => retrieveComponentStyles(COMPONENT_ID, props)};
 `;
 
 StyledTooltip.defaultProps = {
   hasArrow: true,
```

Comment:
Shouldn't have to default with this opinion – let the higher-abstraction component decide.

Reply:
Moved to the element level.